source/operations/install-deploy-manage/upgrade-minio-deployment.rst

```diff
@@ -10,6 +10,10 @@ Upgrade a MinIO Deployment
    :local:
    :depth: 2
 
+.. important::
+
+   For deployments using :ref:`AD/LDAP <minio-ldap-config-settings>` for external authentication upgrading from a version older than :minio-release:`RELEASE.2024-03-30T09-41-56Z` **must** read through the release notes for :minio-release:`RELEASE.2024-04-18T19-09-19Z` before starting this procedure.
```

Comment:
possibly you intended this?
```suggestion
   For deployments using :ref:`AD/LDAP <minio-ldap-config-settings>` for external authentication and upgrading from a version older than :minio-release:`RELEASE.2024-03-30T09-41-56Z`: you **must** read through the release notes for :minio-release:`RELEASE.2024-04-18T19-09-19Z` before starting this procedure.
```

Reply:
also for the same text in upgrade tenant

Reply:
```suggestion
   For deployments older than :minio-release:`RELEASE.2024-03-30T09-41-56Z` running with :ref:`AD/LDAP <minio-ldap-config-settings>` enabled, you **must** read through the release notes for :minio-release:`RELEASE.2024-04-18T19-09-19Z` before starting this procedure.
```

ThoughtS?

Reply:
Yes!